### ISSUE TITLE
Error with Django 2.1 on Python 3.7

### ISSUE THREAD

**Issue by benldietchef** [citusdata/django-multitenant#41]
```
./dev.sh django-admin sqlmigrate core 0001
Starting machinelabsbackend_memcached_1 ... done
Reading config from /volumes/environment/01_environment.json
Reading config from /volumes/environment/02_secrets.json
Traceback (most recent call last):
  File "/usr/local/bin/django-admin", line 11, in <module>
    sys.exit(execute_from_command_line())
  File "/usr/local/lib/python3.7/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.7/site-packages/django/core/management/__init__.py", line 375, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python3.7/site-packages/django/core/management/base.py", line 316, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/local/lib/python3.7/site-packages/django/core/management/commands/sqlmigrate.py", line 29, in execute
    return super().execute(*args, **options)
  File "/usr/local/lib/python3.7/site-packages/django/core/management/base.py", line 353, in execute
    output = self.handle(*args, **options)
  File "/usr/local/lib/python3.7/site-packages/django/core/management/commands/sqlmigrate.py", line 58, in handle
    sql_statements = executor.collect_sql(plan)
  File "/usr/local/lib/python3.7/site-packages/django/db/migrations/executor.py", line 225, in collect_sql
    state = migration.apply(state, schema_editor, collect_sql=True)
  File "/usr/local/lib/python3.7/site-packages/django/db/migrations/migration.py", line 124, in apply
    operation.database_forwards(self.app_label, schema_editor, old_state, project_state)
  File "/usr/local/lib/python3.7/site-packages/django/db/migrations/operations/models.py", line 91, in database_forwards
    schema_editor.create_model(model)
  File "/usr/local/lib/python3.7/site-packages/django/db/backends/base/schema.py", line 285, in create_model
    self.deferred_sql.append(self._create_fk_sql(model, field, "_fk_%(to_table)s_%(to_column)s"))
  File "/usr/local/lib/python3.7/site-packages/django_multitenant/backends/postgresql/base.py", line 58, in _create_fk_sql
    "name": self.quote_name(self._create_index_name(model, from_columns, suffix=suffix)),
  File "/usr/local/lib/python3.7/site-packages/django/db/backends/base/schema.py", line 877, in _create_index_name
    _, table_name = split_identifier(table_name)
  File "/usr/local/lib/python3.7/site-packages/django/db/backends/utils.py", line 204, in split_identifier
    namespace, name = identifier.split('"."')
AttributeError: type object 'CoreLTVCohort' has no attribute 'split'
```

**Comment by benldietchef** [comment]
This was with django_multitenant==2.0.0 installed through pip

**Comment by louiseGrandjonc** [comment]
Hi @benldietchef, a fix is being reviewed, if you want to use the branch here it is :) https://github.com/citusdata/django-multitenant/pull/42

**Comment by louiseGrandjonc** [comment]
The new version has been released on pypi django_multitenant==2.0.1 with the fix
